docs/installation.rst

```diff
@@ -3,7 +3,7 @@ Installation
 
 Getting Python
 --------------
-Cedar-solve is written for Python 3.7 or later (and therefore runs on almost any platform) and should
+Cedar-solve is written for Python 3.8 or later (and therefore runs on almost any platform) and should
```

Comment:
this was changed due to `math.comb` not existing in 3.7, the `pyproject.toml` file specifies `>=3.8`